tutorials/3_transactions.md

```diff
@@ -241,28 +302,59 @@ sender [uint8]: 3
 blocknum [uint256]: 0
 timestamp [uint256]: 0
 next_call [uint8]: 1
-[Calling claim on contract_1]
+[Calling claim() on contract_1]
 [Call successful]
 sender [uint8]: 3
 blocknum [uint256]: 0
 timestamp [uint256]: 0
 next_call [uint8]: 2
-[Calling open on contract_1]
+[Calling open() on contract_1]
 [Call successful]
 sender [uint8]: 3
 blocknum [uint256]: 0
 timestamp [uint256]: 0
 next_call [uint8]: 4
-[Calling deposit on contract_1]
+[Calling deposit() on contract_1]
 value [uint256]: 1
 [Call successful]
 [sea] __VERIFIER_error was executed
 ```
 
-If we follow through this trace, we can see a single client `claim`:
+At first this counterexample might seem surprising. No client should be able to
+call `open()` directly. However, this is precisely what happens. To figure out
+why, we can log the owner of `contract_1` during each iteration of the
+transaction loop. After making these changes we can rerun:
+
+  * `make witness`
+  * `./witness`
+
+To obtain the following trace:
 
-  1. Acquire ownership of `fund` through `claim`.
-  2. Transition `fund` to an accepting state through `open`.
-  3. Move a single Ether into `fund` through `deposit`.
+```
+[Constructing contract_0 and children]
+sender [uint8]: 3
+[Entering transaction loop]
+=> owner of contract_1: 1
+sender [uint8]: 3
+next_call [uint8]: 1
+[Calling claim() on contract_1]
+[Call successful]
+=> owner of contract_1: 3
+sender [uint8]: 3
+next_call [uint8]: 2
+[Calling open() on contract_1]
+[Call successful]
+=> owner of contract_1: 3
+sender [uint8]: 3
+next_call [uint8]: 4
+[Calling deposit() on contract_1]
+value [uint256]: 1
+[Call successful]
+[sea] __VERIFIER_error was executed
+```
 
-This is the counterexample we suspected, and it is not spurious.
+We can now see how this is possible. When `Manager` is constructed, it creates a
+new fund, whose owner is `Manager`. This aligns with the first owner. Client 3
+then `claim()` and takes ownership. This aligns with the second owner. From
+here, client 3 is free to `open()` the contract, and then `deposit()` a single
+Ether. The balance of `Fund` has changed without a call to `openFund()`.
```

Comment:
Maybe it would be good to somehow summarize each blog post at the end (2-3 sentences about what we "learned") and provide a pointer to what will be covered in subsequent posts.